src/test/resources/com/puppycrawl/tools/checkstyle/grammar/comments/InputFullOfSinglelineComments.java

```diff
@@ -1,64 +1,70 @@
-//0
-package//1
-com//2
-.//3
-puppycrawl//4
-.//5
-tools//6
-.//7
-checkstyle.grammar//8
-.//9
-comments//10
-//11
-//12
-;//13
-//14
-public//15
-class//16
-InputFullOfSinglelineComments//17
-{//18
-public//19
-static//20
-void//21
-main//22
-(//23
-String//24
-[//25
-]//26
-args//27
-)//28
-{//29
-for//30
-(//31
-int//32
-i//33
-=//34
-0//35
-;//36
-i//37
-<=//38
-10//39
-;//40
-i++//41
-)//42
-{//43
-String//44
-line//45
-=//46
-i//47
-+//48
-""//49
-;//50
+/*
+com.puppycrawl.tools.checkstyle.grammar.comments.AllSinglelineCommentsTest$SinglelineCommentListenerCheck
```

Comment:
Note the `$`, this is how we can reference a static inner class for usage in `Class#forName` in `InlineConfigParser`.